lua/nvim-treesitter/highlight.lua

```diff
@@ -81,19 +81,14 @@ function M.attach(bufnr, lang)
   local parser = parsers.get_parser(bufnr, lang)
   local config = configs.get_module('highlight')
 
-  if config.use_languagetree then
-    local ltree = require'nvim-treesitter.languagetree'
-    ltree.new(bufnr, lang)
-  else
-    for k, v in pairs(config.custom_captures) do
-      hlmap[k] = v
-    end
-
-    local query = queries.get_query(lang, "highlights")
-    if not query then return end
-
-    M.highlighters[bufnr] = ts.highlighter.new(parser, query)
+  for k, v in pairs(config.custom_captures) do
+    hlmap[k] = v
   end
+
+  local query = queries.get_query(lang, "highlights")
+  if not query then return end
+
+  M.highlighters[bufnr] = ts.highlighter.new(parser, query)
```

Comment:
The second argument to this is now an options object... You actually don't need to provide the query at all. It will get looked up by nvim lazily.

Reply:
The process here is just mimicking what is happening upstream, I don't think it should bother that much to have that here too ?